.gitignore

```diff
@@ -6,4 +6,3 @@
 .idea
 npm-debug.log
 node_modules
-dist
```

Comment:
Can you please add this line again? and let's restore the package-lock.

I think we are good only with the changes on src/index.js